testsuite/Rakefile

```diff
@@ -147,11 +153,14 @@ namespace :utils do
     clients.reject! { |client| client.include? 'ssh' }
     clients.reject! { |client| (client.include? 'buildhost') || (client.include? 'terminal') }
     clients.each do |client|
+      # disable flush option for s390 minions
+      flush = client.include?('s390') ? 'false' : 'true'
```

Comment:
Nitpick: if `include?` returns a boolean, can't we just use that value instead of strings ?

```ruby
flush = !client.include?('s390')
```